mpt_api_client/models/__init__.py

```
@@ -1,6 +1,6 @@
-from mpt_api_client.models.collection import Collection
 from mpt_api_client.models.file_model import FileModel
 from mpt_api_client.models.meta import Meta, Pagination
 from mpt_api_client.models.model import Model, ResourceData
+from mpt_api_client.models.model_collection import ModelCollection
 
-__all__ = ["Collection", "FileModel", "Meta", "Model", "Pagination", "ResourceData"]  # noqa: WPS410
+__all__ = ["FileModel", "Meta", "Model", "ModelCollection", "Pagination", "ResourceData"]  # noqa: WPS410
```

Ruff (0.15.10)

[warning] Invalid rule code in `# noqa`: WPS410

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)